tests/interactive/test_get_all_available_passes.py

```diff
@@ -82,25 +81,15 @@ def test_get_all_available_passes():
         (
             AvailablePass(
                 display_name="bc",
-                module_pass=BCPass,
-                pass_spec=None,
+                module_pass=BCPass(),
```

Comment:
nit: It looks like AvailablePass can still be built with a type?

Reply:
Yep, this is used when "Condense Mode" is not on, and for mlir-opt pass as we want to present the pass as an option to the user without instantiating it